File: clients/android-fireback/app/src/main/java/com/fireback/modules/workspaces/TokenEntity.java
package com.fireback.modules.workspaces;
import com.fireback.modules.workspaces.OkayResponseDto;
import com.google.gson.Gson;
import com.fireback.JsonSerializable;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;
public class TokenEntity extends JsonSerializable {
    public UserEntity user;
    public String validUntil;
    public static class VM extends ViewModel {
    // upper: User user
    private MutableLiveData< UserEntity > user = new MutableLiveData<>();
    public MutableLiveData< UserEntity > getUser() {
        return user;
    }
    public void setUser( UserEntity  v) {
        user.setValue(v);
    }
    // upper: ValidUntil validUntil
    private MutableLiveData< String > validUntil = new MutableLiveData<>();
    public MutableLiveData< String > getValidUntil() {
        return validUntil;
    }
    public void setValidUntil( String  v) {
        validUntil.setValue(v);
    }
    }
}

File: modules/fireback/codegen/ios/firebackios/modules/workspaces/TokenEntity.swift
class TokenEntity : Codable {
    private var _user: UserEntity?
    var `User`: UserEntity? {
        set { _user = newValue }
        get { return _user }
    }
    private var _validUntil: String?
    var `ValidUntil`: String? {
        set { _validUntil = newValue }
        get { return _validUntil }
    }
}